src/theme.ts

```diff
@@ -1,14 +1,41 @@
 /* eslint-disable @mll-lab/no-color-literals */
 
+import {
+  red,
+  volcano,
+  orange,
+  gold,
+  yellow,
+  lime,
+  green,
+  cyan,
+  blue,
+  geekblue,
+  purple,
+  magenta,
+  grey,
+} from '@ant-design/colors';
 import { useContext } from 'react';
 import { ThemeContext } from 'styled-components';
 
-/**
- * We omit special purpose colors:
- * - 'white': Brightest possible
- * - 'black': Darkest possible
- * - 'transparent': No color
- */
+export const MLL_PALETTE = {
+  red: red.primary,
+  volcano: volcano.primary,
+  orange: orange.primary,
+  gold: gold.primary,
+  yellow: yellow.primary,
+  lime: lime.primary,
+  green: green.primary,
+  cyan: cyan.primary,
+  blue: blue.primary,
+  geekblue: geekblue.primary,
+  purple: purple.primary,
+  magenta: magenta.primary,
+  grey: grey.primary,
+  white: '#ffffff',
+  black: '#000000',
+};
+
 export const MLL_THEME = {
   borderColor: '#50a0d0',
```

Comment:
We should define those in terms of the palette. This has the benefit of forcing us to either:
- increase the consistency of the theme
- discover missing colors and adding them to the palette

I would err on the side of fewer colors and a small palette.

Reply:
I was able to merge some colors and I've defined grey tints and specific colors for the remaining colors.
@spawnia let me know if and how you would handle the other colors.